docs/providers/aws/examples/hello-world/go/README.md

```diff
@@ -0,0 +1,60 @@
+<!--
```

Comment:
For `aws-go-dep` example, it only works if we are within a `$GOPATH/src` folder.

I'm just wondering if maybe we should document it somewhere and refer to https://golang.org/doc/code.html#GOPATH?

Reply:
true, these templates and docs are aimed at first time users. It would be better to make them more beginner-friendly. Will update